homeassistant/generated/config_flows.py

```diff
@@ -29,6 +29,7 @@
         "airthings",
         "airthings_ble",
         "airtouch4",
+        "airtouch5",
```

Comment:
Are they related?

Reply:
They are similar systems made by the same manufacturer. They have different protocols and behaviours.